types/mocks.d.ts

```diff
@@ -3,6 +3,7 @@ declare module "mac-ca"
 declare module "win-ca"
 declare module "@hapi/call"
 declare module "@hapi/subtext"
+declare module "string.ify"
```

Comment:
I would prefer avoid modules without types nowadays without strong reason to include it.